tensorflow/python/ops/rnn.py

```diff
@@ -73,6 +73,7 @@ def rnn(cell, inputs, initial_state=None, dtype=None,
   """Creates a recurrent neural network specified by RNNCell `cell`.
 
   The simplest form of RNN network generated is:
+
```

Comment:
When I saw https://www.tensorflow.org/versions/r0.9/api_docs/python/nn.html#rnn, this line seems like just one line not a code. 
I think this line should do code block.

According to these examples, just adding newline can be made code block in markdown docs
ex)
https://www.tensorflow.org/versions/r0.9/api_docs/python/nn.html#embedding_lookup_sparse
https://github.com/tensorflow/tensorflow/blob/ac90ecb08d80fc92147fdb3ed852fbb1ddbecf5f/tensorflow/python/ops/embedding_ops.py#L220

https://www.tensorflow.org/versions/r0.9/api_docs/python/nn.html#separable_conv2d
https://github.com/tensorflow/tensorflow/blob/8b74a567875926fa5d9da3614f193ab8e5fc9653/tensorflow/python/ops/nn.py#L595